test/sqllogictest/tpch.slt

```diff
@@ -1443,7 +1443,7 @@ WHERE
 | | implementation = DeltaQuery
 | |   delta %0 %1.(#0)
 | |   delta %1 %0.(#1)
-| Filter (((((((((#22 = "LG BOX") || (#22 = "LG CASE")) || (#22 = "LG PACK")) || (#22 = "LG PKG")) && (#19 = "Brand#34")) && (#4 >= 20)) && (#4 <= 30)) && (#21 <= 15)) || (((((((((#22 = "SM BOX") || (#22 = "SM CASE")) || (#22 = "SM PACK")) || (#22 = "SM PKG")) && (#19 = "Brand#12")) && (#4 >= 1)) && (#4 <= 11)) && (#21 <= 5)) || ((((((((#22 = "MED BAG") || (#22 = "MED BOX")) || (#22 = "MED PKG")) || (#22 = "MED PACK")) && (#19 = "Brand#23")) && (#4 >= 10)) && (#4 <= 20)) && (#21 <= 10)))), ((#14 = "AIR") || (#14 = "AIR REG")), (#13 = "DELIVER IN PERSON"), (#21 >= 1)
+| Filter ((((((((((#22 = "LG BOX") || (#22 = "LG PKG")) || (#22 = "LG CASE")) || (#22 = "LG PACK")) && (#19 = "Brand#34")) && (#4 <= 30)) && (#21 <= 15)) && (#4 >= 20)) || ((((((((#22 = "SM BOX") || (#22 = "SM PKG")) || (#22 = "SM CASE")) || (#22 = "SM PACK")) && (#19 = "Brand#12")) && (#4 <= 11)) && (#21 <= 5)) && (#4 >= 1))) || ((((((((#22 = "MED BAG") || (#22 = "MED BOX")) || (#22 = "MED PKG")) || (#22 = "MED PACK")) && (#19 = "Brand#23")) && (#4 <= 20)) && (#21 <= 10)) && (#4 >= 10))), ((#14 = "AIR") || (#14 = "AIR REG")), (#13 = "DELIVER IN PERSON"), (#21 >= 1)
```

Comment:
There is no intention to have this sorted in lexicographical order, right? Just some deterministic order? Before the change, it seems that lexicographical order was used.

Reply:
Just some deterministic order. Before this patch, they were displayed in the order they were in in the original query.

It seems that for comparing string literals we first use the length and we use the byte-level comparison among those of the same length.